avoidance/src/transform_buffer.cpp

```diff
@@ -67,7 +67,7 @@ bool TransformBuffer::getTransform(const std::string& source_frame, const std::s
     return false;
   } else {
     if (iterator->second.back().stamp_ < time) {
-      // "TF Buffer: could not retrieve requested transform from buffer, tf has not yet arrived"
+      print(log_level::debug, "TF Buffer: could not retrieve requested transform from buffer, tf has not yet arrived");
```

Comment:
We usually use ROS_DEBUG streams for printing, maybe we should stay consistent.

Reply:
This just links to the ROS_DEBUG below, so I don't think there's an issue.